system.go

```diff
@@ -69,3 +70,19 @@ func (s *SystemService) GetDiagnosticID() string {
 	// TODO: Consider deprecating/rewriting in favor of just using GetUnsanitizedConfig().
 	return s.api.GetDiagnosticId()
 }
+
+// RequestTrialLicense requests a trial license and installs it in the server.
+// If the server version is lower than 5.36.0, an error is returned.
+//
+// Minimum server version: 5.4
+func (s *SystemService) RequestTrialLicense(requesterID string, users int, termsAccepted, receiveEmailsAccepted bool) error {
+	currentVersion := semver.MustParse(s.api.GetServerVersion())
+	requiredVersion := semver.MustParse("5.36.0")
+
+	if currentVersion.LT(requiredVersion) {
+		return errors.Errorf("current server version is lower than 5.36")
+	}
```

Comment:
Is this how this should be done? Is there any better way? I'm all ears.

Reply:
What happens if we try to invoke `RequestTrialLicense` without a server implementation? Is that error code predictable, and usable without having to explicitly check versions?

Reply:
I have not actually checked, but I think I like having the explicit version check. It signals the intention better than handling an error, which may need the addition of a comment to explain what's going on.